docs/topics/db/models.txt

```diff
@@ -1326,6 +1334,34 @@ containing the new managers and inherit that after the primary base class::
 You probably won't need to do this very often, but, when you do, it's
 possible.
 
+Another use case for proxy models is to represent a subset of the concrete
```

Comment:
I think rather than creating a whole new example here, using different models, we could just change the `NewManager` to show `get_queryset` customization.

I also think it doesn't need _too much_ spelling out. The second documented use case for custom managers is a filtered queryset, so it's not too surprising that a custom manager on a proxy model can also filter the queryset.